docs/CONTRIBUTING.md

```diff
@@ -54,37 +54,36 @@ The sandbox architecture can be broken down into 3 distinct groups.
 
 2. Support Services
 
-   - [OSIE/Hook File Server](https://docs.tinkerbell.org/services/osie/)
+   - [Hook File Server](https://docs.tinkerbell.org/services/osie/)
```

Comment:
I think this change suggests we need to ship Hook docs sometime soon...

Reply:
Yep all the docs need updating re hook.